source/main/assets/tutorial/fr/zettelkasten.md

```diff
@@ -0,0 +1,58 @@
+
+Désormais, ce fichier a un identifiant que vous pouvez utiliser ! Essayez-le — retournez à l'onglet avec le guide « Bienvenue sur Zettlr ! », et tapez `[[` quelque part. Dans la fenêtre d'autocomplétion, choisissez ce fichier et confirmez votre sélection. Ensuite, `Cmd/Ctrl`, cliquez sur ce même lien pour revenir à ce fichier. Vous remarquerez que Zettlr a lancé une nouvelle recherche, mais surtout : vous pouvez voir les résultats de la recherche en surbrillance ! Ceci est utile pour les liens croisés Zettelkasten, mais sera bien sûr également utile lorsque vous ferez des recherches globales.
+
+
+
+## Utiliser les mots-clés (tags) 🏷
+
+Cela dit, faire des liens n'est pas le seul moyen de créer des relations entre les notes. Vous pouvez également utiliser des mots-clés pour cela. Les mots-clés fonctionnent exactement comme les hashtags sur Twitter, donc vous pouvez #créer #des hashtags #comme #vous #voulez ! En cliquant sur `Cmd/Ctrl`, vous lancerez également une recherche et mettrez en évidence tous les fichiers qui contiennent tel ou tel mot-clé.
+
+Il y a aussi un nuage de mots-clés auquel vous pouvez accéder en cliquant sur l'icône « Mots-clés » dans la barre d'outils. Il énumérera tous vos mots-clés et indiquera le nombre de fichiers qui les utilisent. Vous pouvez filtrer et gérer vos tags à partir de là. Alors que les liens Zettelkasten créent des connexions « dures » entre les fichiers, les tags sont une sorte de connexion « floue » entre des contenus connexes et peuvent vous convenir davantage.
+
+
+## Réflexions finales 💭
+
+Nous ne passerons pas en revue toutes les méthodes de travail dans un Zettelkasten, car il existe de nombreux tutoriels qui vous aideront à démarrer. En voici une liste utile :
+
+- [Une première introduction peut être trouvée dans notre documentation](https://docs.zettlr.com/fr/academic/zkn-method/)
+- [Sur le concept du Zettelkasten, lire notre blog](https://zettlr.com/post/what-is-a-zettelkasten)
+- [La page zettelkasten.de (en anglais) contient de nombreux articles sur les Zettelkästen](https://zettelkasten.de/)
+- [Reddit dispose d'un subreddit consacré exclusivement à l'art du Zettelkasten](https://www.reddit.com/r/Zettelkasten)
+
+
+
+Ces tutoriels constitueront d'excellents points de départ pour votre cheminement dans l'art mystérieux de la création d'un Zettelkasten !
+
+
+Une dernière chose encore : comme la méthode de travail des Zettelkästen n'est pas très standardisée, et qu'il existe de nombreuses manières correctes de le faire, Zettlr vous permet de personnaliser entièrement chaque aspect de la méthode des Zettelkästen. Pour commencer, jetez un coup d'œil à [notre documentation sur la façon dont cela fonctionne](https://docs.zettlr.com/en/reference/settings/#zettelkasten) !
+
+**Vous en voulez plus ?** Alors consultez notre guide sur les [[références]] avec Zettlr !
```

Comment:
This link will not work, as there is no file `références.md` in the folder. It should work by either changing this link back to `citing` (to point to the file `citing.md`) OR by renaming the `citing`-file to `références.md` (albeit I would like to double-check that it works, just to be sure!)

The only file name that is required to remain the same is the `welcome.md` as Zettlr expects this file (to open it by default).

Furthermore: Would you mind exporting the Pandoc & LaTeX-Markdown file to PDF using Zettlr (export to this directory)? This is the clever thing about the tutorial showing off the sidebar and how files may look if you export them using Zettlr!

Reply:
Done : `citing.md` became `reference.md ` (as I don't like accents in file names).